modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -102,6 +102,8 @@ As an alternative to `.spec.resourceCustomizations`, you can use `.spec.resource
 To prevent data loss during upgrade to {gitops-title} Operator v1.10.0, ensure that you backup `.spec.resourceCustomization` value if it is used in your Argo CD CRs.
 ==== 
 
+* With this update, support for deprecated Legacy Configuration Management Plugins (CMPs) specified in `argo-cm` or through the Operator `argocd.spec.configManagementPlugins` is removed in Argo CD v2.8. To continue using your legacy plugins, consider migrating them to the new sidecar available in the Operator through `argocd.spec.repo.sidecarContainers`. link:https://issues.redhat.com/browse/GITOPS-3462[GITOPS-3462]
```

Comment:
... support for deprecated Legacy Configuration Management Plugins (CMPs) specified in  the `argo-cm` config map **or** through the Operator `argocd.spec.configManagementPlugins` is removed ...

The "or" here seems ambiguous. Does it mean "either, or"  or can we choose where it is specified, or is it specified in both places? Please check and make changes accordingly.

Reply:
Hello @svghadi , would you please assist in clarifying this ?

Reply:
It's **"either, or"**

Reply:
@ekristova , I have addressed this now.